Orange/widgets/visualize/tests/test_owmap.py

```diff
@@ -67,11 +68,18 @@ def test_coverage(self):
         self.widget.map.selected_area(90, 180, -90, -180)
         self.widget.map.set_map_provider(next(iter(self.widget.TILE_PROVIDERS.values())))
         self.widget.map.set_clustering(True)
+        self.widget.map.set_clustering(False)
         self.widget.map.set_jittering(5)
         self.widget.map.set_marker_color('latitude')
         self.widget.map.set_marker_label('latitude')
         self.widget.map.set_marker_shape('foo')
         self.widget.map.set_marker_size('latitude')
+        self.process_events(
+            until=lambda start_time=time.clock(): time.clock() - start_time > .2)
+        self.widget.map.set_marker_color(None)
+        self.widget.map.set_marker_label(None)
+        self.widget.map.set_marker_shape(None)
+        self.widget.map.set_marker_size(None)
```

Comment:
Duplicated code. Why not:
´´´ python
        def set_marker(color, label, shape, size):
            self.widget.map.set_marker_color(color)
            self.widget.map.set_marker_label(label)
            self.widget.map.set_marker_shape(shape)
            self.widget.map.set_marker_size(size)
´´´
And then:
```python
set_marker('latitude', 'latitude', 'foo', 'latitude')
´´´
and
```python
set_marker(*[None] * 4)
´´´

Reply:
In your opinion, what's the line tally in both cases?